pkg/component/shared/istio_test.go

```diff
@@ -58,6 +64,7 @@ func createIstio(testValues istioTestValues) istio.Interface {
 	))
 
 	istio, err := NewIstio(
+		context.TODO(),
```

Comment:
```suggestion
		context.Background(),
```

Reply:
Replaced all `context.TODO()` in the test file as suggested.